ci/requirements-2.7_DOC_BUILD.run

```diff
@@ -14,3 +14,4 @@ sqlalchemy
 numexpr
 bottleneck
 statsmodels
+xarray
```

Comment:
If it is only for one line of code in the whatsnew, I don't know if it is worth adding it as a doc build dependency, instead of just using a literal `code-block`